pythonFiles/jedi/api/__init__.py

```diff
@@ -87,7 +87,7 @@ def __init__(self, source=None, line=None, column=None, path=None,
         self.path = None if path is None else os.path.abspath(path)
 
         if source is None:
-            with open(path) as f:
+            with open(path, encoding=encoding) as f:
```

Comment:
@IgorNovozhilov how about adding a try..except.
If the above code doesn't work, then try opening with the encoding.
I prefer that approach as that will leave it backward compatible (ensuring we don't introduce any other issues).
If you can make that change I can accept this and create a build soon.